header_test.sh

```diff
@@ -0,0 +1,19 @@
+# !/usr/bin/env bash
+
+# Collate files
+python_files="$(find ./cli ./smarts ./benchmark ./envision -name '*.py')"
```

Comment:
Note didn't check ultra directory since it doesn't exist yet

Reply:
Issue created out of it: https://github.com/huawei-noah/SMARTS/issues/416